### ISSUE TITLE
What's the best way to restrict logging?

### ISSUE THREAD

**Issue by robneu** [KingYes/wordpress-aryo-activity-log#85]
Basically, we don't want to track all of the data that is being tracked by default because it's not all useful to us and we'd rather keep the logs as lightweight as possible.

I had a quick read-through of the plugin and I found a few different areas that seem like they're designed to allow developers to limit what is and isn't tracked, but I'm not 100% sure what the best way to approach this is.

Could you provide any example code or just point me in the right direction for what needs to be filtered within the plugin to accomplish this?

**Comment by KingYes** [comment]
For now you can't exclude some data.
By the way, you don't have any issue with this data. We are store all the data in separator table, so you don't get any performance issue.

**Comment by KingYes** [comment]
Good point. I'll check this out..

Thanks

**Comment by thedeviousdev** [comment]
+1, I have a feature on my site that is constantly querying an outside database to push orders through, I don't necessarily want to track that as I can view that through another API.

**Comment by amooreTO** [comment]
I agree. It would be great to disable some of the logging that we don't need or want to capture.

**Comment by StefsterNYC** [comment]
@amooreTO I ended up writing a MU plugin that you can add to. This stops all the shop_order post type action logs. Place it in your MU plugins folder. Build on to it based on what logs you don't want. 


```
<?php
/*
Plugin Name: Disable Aryo Logging for WooCommerce Orders
Author: Stef Serafin from SerafinTech
Description: Prevents Aryo Activity Log from logging shop_order entries.
*/

add_filter( 'aryo_activity_log_prevent_log', function( $prevent, $context ) {
    if ( isset( $context['post_type'] ) && $context['post_type'] === 'shop_order' ) {
        return true;
    }
    return $prevent;
}, 10, 2 );
```
